src/Plugin_Toggler.php

```diff
@@ -46,6 +65,10 @@ public function init() {
 			return;
 		}
 
+		if ( ! $this->options['plugin_toggler'] ) {
```

Comment:
Suggest using `empty()` check, to return false when not set, instead of an undefined index warning.